progress-bar.dust

```diff
@@ -0,0 +1,24 @@
+<div class="c-progress-bar c--{state} {?isRadial}c--radial{/isRadial} {class}" {?id}id="{.}"{/id} role="progressbar" aria-valuenow="{value}" aria-valuemin="0" aria-valuemax="1" aria-valuetext="{label}" data-adaptivejs-component="stencil-progress-bar">
+
+    {?isRadial}
+        {@c-progress-bar__radial/}
+    {/isRadial}
+
+    <div class="c-progress-bar__middle">
+        <div class="c-progress-bar__label">{label}</div>
+
+        {^isRadial}
+        <div class="c-progress-bar__linear">
+            <div class="c-progress-bar__linear-background">
```

Comment:
Is this container necessary? Couldn't you just apply the background and positioning properties on the `.c-progress-bar__linear` container?